packages/lib/attributeClass/service.ts

```diff
@@ -7,64 +7,89 @@ import {
   TAttributeClassUpdateInput,
   ZAttributeClassUpdateInput,
   TAttributeClassType,
-} from "@formbricks/types/attributeClasses";
-import { ZId } from "@formbricks/types/environment";
+  ZAttributeClassType,
+} from "@formbricks/types/v1/attributeClasses";
+import { ZId } from "@formbricks/types/v1/environment";
 import { validateInputs } from "../utils/validate";
-import { DatabaseError } from "@formbricks/types/errors";
-import { revalidateTag, unstable_cache } from "next/cache";
+import { DatabaseError } from "@formbricks/types/v1/errors";
```

Comment:
will throw a build error on newest branch, types are no longer in the v1 folder. Please remove the v1 😊

Reply:
Sounds good.